### PR TITLE
Update AttachDocumentsforViewers

### PR DESCRIPTION
**Pull request by hernandezfo**
Update AttachDocumentsforViewers. Issues Fixed. New version 0.0.2

[AttachDocumentsforViewers-0.0.2.zip](https://github.com/kanboard/website/files/2766601/AttachDocumentsforViewers-0.0.2.zip)

[SourceCode-AttachDocumentsforViewers-0.0.2.zip](https://github.com/kanboard/website/files/2766602/SourceCode-AttachDocumentsforViewers-0.0.2.zip)

### CODE REVIEW

**Comment by creecros** [comment]
Your attached zips still show version 0.0.1 in Plugin.php.

You need to place these in your repo, either in the repo file directory or as a [release](https://github.com/hernandezfo/AttachDocumentsforViewers/releases). Then update the download link in the json, as well.

```php
 "download": "https://github.com/hernandezfo/AttachDocumentsforViewers/releases/download/0.0.2/AttachDocumentsforViewers-0.0.2.zip",
```

**Comment by hernandezfo** [comment]
Repository and json updated

**Comment by creecros** [comment]
That's a dead download link until you draft new release with the 0.0.2 tag.

File: plugins.json [hunk 1]
```diff
@@ -889,13 +889,13 @@
...
         "title": "Attach Documents for Project Viewers",
-        "version": "0.0.1",
+        "version": "0.0.2",
         "author": "Franklin Hernandez",
         "license": "MIT",
         "description": "This plugin will allow Project Viewers to attach documents to tasks.",
         "homepage": "https://github.com/hernandezfo/AttachDocumentsforViewers",
         "readme": "https://raw.githubusercontent.com/hernandezfo/AttachDocumentsforViewers/master/README.md",
-        "download": "https://github.com/hernandezfo/AttachDocumentsforViewers/releases/download/0.0.1/AttachDocumentsforViewers-0.0.1.zip",
+        "download": "https://github.com/hernandezfo/AttachDocumentsforViewers/releases/download/0.0.2/AttachDocumentsforViewers-0.0.2.zip",
```
Comment (fguillot) [inline plugins.json:898]: This link returns a 404.

**Review by fguillot — CHANGES_REQUESTED** [review]

**Comment by fguillot** [comment]
Closing stale PR after 1 month of inactivity.

**Pull request closed without merging**